src/components/Products/ProductsListItem.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="product-list-item">
+    <div
+      class="product-list-item__picture"
+      v-bind:style="{ backgroundImage: 'url(' + imgUrl + ')' }"
+    ></div>
+    <div class="product-list-item__description">
+      <span class="product-list-item__description__title" v-if="name.length < 20">{{ name }} </span>
+      <span class="product-list-item__description__title" v-if="name.length >= 20">{{ name.substring(0,14).concat("..") }} </span>
```

Comment:
you can use v-else in regard to previous line
just change v-if="name.length >= 20" to v-else